plugins/aeon_to_lobby/lobbytrack.py

```
@@ -0,0 +1,31 @@
+import logging
+import requests
+
+from airflow.models import Variable
+
+lobby_url = Variable.get("LOBBY_URL")
+lobby_key = Variable.get("LOBBY_KEY")
+
+lobby_headers = {
+    "Authorization": lobby_key,
+    "Content-type": "application/json",
+    "Accept": "application/json",
+}
+
+
+def lobby_post(*args, **kwargs):
+    task_instance = kwargs["task_instance"]
+    lobby_users = task_instance.xcom_pull(
+        key="return_value", task_ids="transform_aeon_data_to_lobby_json"
+    )
+
+    for user in lobby_users:
+        response = requests.post(lobby_url, headers=lobby_headers, json=user)
+
+    if response.status_code != 200:
+        logging.error(
+            "lobbytrack api rsponded with:" f"{response.status_code}, {response.text}"
+        )
+        return None
```

On further reflection, should we raise an error to fail the task instead of just returning None? Maybe replace this line with `response.raise_for_status()`?